testing_data/model_config.json

```diff
@@ -1,39 +1,36 @@
 {
```

Comment:
Not relevant, new used file is in ivadomed/data-testing

Reply:
I know I am repeating myself but why not completely deleting testing_data?

Reply:
It will be done after this PR. See #391